tensorboard/plugins/beholder/beholder_plugin.py

```diff
@@ -137,8 +135,7 @@ def _serve_change_config(self, request):
   def _serve_section_info(self, request):
     path = '{}/{}'.format(
         self.PLUGIN_LOGDIR, shared_config.SECTION_INFO_FILENAME)
-    info = file_system_tools.read_pickle(path, default=self.most_recent_info)
-    self.most_recent_info = info
+    info = file_system_tools.read_pickle(path, default=DEFAULT_INFO)
```

Comment:
Again I'm not sure why we're removing the "cache last data shown" behavior.  It's not really necessary to fix this bug, since we could still keep most_recent_info and then just default to None here, and then

```py
if info is None:
  info = self.most_recent_info if self.most_recent_info is not None else DEFAULT_INFO
```

Reply:
Could we preserve the caching behavior here too if we're doing it for the frame?  Right now __init__ is storing most_recent_info but it's never being used.  I think just reverting the delta at these lines would suffice.

Reply:
Done